components/lists/Lists.css

```diff
@@ -0,0 +1,61 @@
+.list {
+  margin: 0 1.5rem;
+}
+
+.stack-list {
+  margin: 6px auto;
+  padding: 5px;
+  background-color: beige;
+  max-width: 350px;
```

Comment:
use can use rem instead of px

Reply:
Hey sure, as Vikram suggested, I'm using px for value less than `10`, but for width I will change. Thanks.